pkg/config/config_template.yaml

```diff
@@ -1460,3 +1460,41 @@ api_key:
 # cluster_name: <CLUSTER_IDENTIFIER>
 
 {{ end -}}
+{{- if .CloudFoundryBBS }}
+#######################################################
+## Cloud Foundry BBS Configuration for Autodiscovery ##
+#######################################################
+
+## @param cloud_foundry_bbs - custom object - optional
+## This section configures how Cluster Agent accesses BBS API to gather information
+## necessary for autodiscovery on BBS-based Cloud Foundry deployments.
+#
+# cloud_foundry_bbs:
+
+  ## @param url - string - optional - default: https://bbs.service.cf.internal:8889
+  ## URL of the BBS API.
+  #
+  # url: https://bbs.service.cf.internal:8889
+
+  ## @param poll_interval - integer - optional - default: 15
+  ## How often to refresh data from BBS API, in seconds. Values lower than 10 might influence
```

Comment:
`Interval in seconds, the BBS API will be called to refresh collected data about ... ` sounds weird. @nmuesch may have a better sentence.